### PR TITLE
update copy_unique README.md

### PR DESCRIPTION
**Pull request by mawillcockson**

### CODE REVIEW

**Comment by mw-review** [comment]
This needs to be rebased onto the most recent commit

File: copy_unique/README.md [hunk 1]
```diff
@@ -4,6 +4,25 @@ Copies unique files from one directory to another.
...
+This tool is incredibly slow. [`rsync`][] is a better tool.
+
+## Installation
+
+It's suggested to use [`pipx`][]:
+
+```sh
+python -m pip install --user --upgrade pip setuptools wheel pipx
+python -m pipx ensurepath
+python -m pipx install 'git+https://github.com/mawillcockson/utilities#egg=copy_unique&subdirectory=copy_unique'
```
Comment (mw-review) [inline copy_unique/README.md]: If this tool is using [`poetry`](https://python-poetry.org/), would `pip` be able to install this project properly?

File: copy_unique/README.md [hunk 2]
```diff
@@ -4,6 +4,25 @@ Copies unique files from one directory to another.
...
+```
+
+Plain [`pip`][] should be:
+
+```sh
+python -m pip install --user --upgrade pip setuptools wheel
+python -m pip install --user 'git+https://github.com/mawillcockson/utilities#egg=copy_unique&subdirectory=copy_unique'
```
Comment (mw-review) [inline copy_unique/README.md]: Same comment here

**Review by mw-review — CHANGES_REQUESTED** [review]
The installation instructions need to be updated.

**Comment by mawillcockson** [comment]
The installation instructions can be added once ci/cd lands.

**Review by mw-review — APPROVED** [review]

File: copy_unique/README.md (hunk 1 above)
Comment (mawillcockson) [reply copy_unique/README.md]: According to [the `pip` documentation](https://github.com/pypa/pip/blob/6c09bdd7c0aff4bb48d9524b9e46d94d1830e9da/docs/html/reference/pip.rst#build-system-interface), the `pyproject.toml` will be referenced. For this project, that means `pip` will use `poetry`.

**Pull request merged**